src_c/circle.c

```diff
@@ -262,11 +262,36 @@ pg_circle_collidepoint(pgCircleObject *self, PyObject *const *args,
                  "collidepoint requires a point or PointLike object");
 }
 
+static PyObject *
+pg_circle_as_rect(pgCircleObject *self, PyObject *_null)
```

Comment:
you can remove `PyObject *_null` as it is `METH_NOARGS` normally

Reply:
It is still of type PyCFunction, even under NoArgs, correct?

we cannot do function pointer casting in pygame as it breaks WASM.

Reply:
starbuck is right here